programs/openbook-v2/src/state/market.rs

```diff
@@ -218,6 +219,24 @@ impl Market {
             / I80F48::from_num(self.base_lot_size)
     }
 
+    /// Convert the quantity from quote lots to native quantity (e.g. 5 SOL, 0.3 USDC, etc)
+    pub fn quote_lot_to_native_quantity(&self, quote_lots: i64) -> f64 {
+        let quote_lot_size = self.quote_lot_size as f64;
+        let quote_decimals = self.quote_decimals as u32;
+        let quote_units = quote_lots as f64 * quote_lot_size;
+        let quantity = quote_units / 10_i64.pow(quote_decimals) as f64;
+        quantity
+    }
```

Comment:
imo better to return fixed point here and let users convert to float themselves

Reply:
Great point! Will fix it soon!